data/_profiles/PetStore.yaml

```diff
@@ -0,0 +1,12 @@
+---
+name: Pet store
+description: "a demo schema"
+
+profileTags: Petstore
+
+demoData: _demodata/applications/petstore
+
+settings: _settings/petstore
```

Comment:
this is a weird place. I would put the settings inside the specific/petstore so things that belong together are close together.